frontend/src/employee-frontend/components/finance-basics/ServiceNeedItem.tsx

```diff
@@ -0,0 +1,34 @@
+// SPDX-FileCopyrightText: 2017-2022 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import React, { useCallback, useState } from 'react'
+
+import { CollapsibleContentArea } from 'lib-components/layout/Container'
+import { H4 } from 'lib-components/typography'
+
+export type PlacementTypeItemProps = {
+  serviceNeed: string
+}
+export default React.memo(function ServiceNeedItem({
+  serviceNeed
+}: PlacementTypeItemProps) {
+  const [open, setOpen] = useState(false)
+  const toggleOpen = useCallback(() => setOpen((isOpen) => !isOpen), [setOpen])
+
+  return (
+    <>
+      <CollapsibleContentArea
+        opaque
+        title={<H4>{serviceNeed}</H4>}
+        open={open}
+        toggleOpen={toggleOpen}
+        paddingHorizontal="0"
+        paddingVertical="0"
+      >
+        <>{/* TODO: voucher value listing here */}</>
+      </CollapsibleContentArea>
+      <div className="separator small" />
```

Comment:
While there seems to be some existing css definition for `.separator`, I don't see any for `.small` and probably this should anyway be replaced by a styled component.

Reply:
Guess that should indeed be just separator if it's not large. I modeled this usage from FeeThresholdsItem in the same context, but what would be a better way to do this? (Looking at git blame this seems to be so 2022...)

Reply:
I would at least check if the HorizontalLine under lib-components would work, name sounds promising. If not, there also seems to be five different `const Separator = styled.div...` components and a bunch of `styled.hr` components so maybe some of them could be made reusable. With this classname approach the biggest issue is that it's quite difficult to find out where that css is defined, so any suitable styled component, whether imported from somewhere or defined locally in this file, would be better.

Reply:
HorizontalLine seems to do the trick, thanks.